Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -5670,6 +5670,36 @@ static VisiblePositionRange constrainRangeToSelection(const VisiblePositionRange
     });
 }
 
+void WebPage::didSwallowClickEvent(const PlatformMouseEvent& event, Node& node)
+{
+    if (!m_userIsInteracting)
+        return;
+
+    if (event.type() != PlatformEventType::MouseReleased)
+        return;
+
+    if (event.syntheticClickType() != SyntheticClickType::NoTap)
+        return;
+
+    RefPtr element = dynamicDowncast<Element>(node);
+    if (!element)
+        return;
+
+    Ref document = node.document();
+    if (!document->quirks().shouldSynthesizeTouchEventsAfterNonSyntheticClick(node))
+        return;
+
+    bool isReplaced = false;
+    auto bounds = node.absoluteBoundingRect(&isReplaced);
```

Comment:
Sad that we have to make this accommodation... maybe someone should add this to `Node`

```cpp
LayoutRect Node::absoluteBoundingRect()
{
    bool unusedIsReplaced;
    return absoluteBoundingRect(&unusedIsReplaced);
}
```

Reply:
Yeah, that — or add a null check for the `bool*` outparam (with a `nullptr` default argument).

Might do that in a followup (since a bunch of other unrelated call sites could adopt it)